src/data/providers/balancer-api/modules/pool-state/index.ts

```diff
@@ -2,10 +2,12 @@ import { BalancerApiClient } from '../../client';
 import { PoolState, PoolStateWithBalances } from '../../../../../entities';
 import { mapPoolType } from '../../../../../utils/poolTypeMapper';
 
+import { API_CHAIN_NAMES } from '../../../../../utils/constants';
+
 export class Pools {
     readonly poolStateQuery = `
-    query GetPool($id: String!) {
-      poolGetPool(id:$id) {
+    query poolGetPool($id: String!, $chain: GqlChain!) {
```

Comment:
I didn't realise the API had changed to require the chain field. Seems nicely handled 👍